6th Small Improvements

zumbak04 opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
# Changelog:
- Jaina and Kael'thas decide who to marry, not Antonidas due to being their liege in the year 603 bookmark.
- Union members don't join civil wars.
- Arthas and Jaina prefer not to marry anyone for the first 5 years (to give them time).
- Becoming undead stops pregnancy.
- Demons, undead, amphibious and reptilian races can't be murdered by poison schemes.
- Dragons, harpies can't be murdered by fall accident schemes.
- Amphibious races, nagas can't be murdered by boat accident schemes.
- To stay in the Union, Union Members have to pay a part of their income, levies, and suffer from high maintenance of MaAs.
- The Union can't be created 10 years after it has been destroyed. Added the Break Union debug interaction.
- The Shatter Realm CB can be used against union leaders to destroy their unions.
- Introduced the Crisis game concept that includes the Scourge, Legion, Horde, etc. (it's just a text, read it, suggest changes).
- Added a game concept of the Shatter Realm CB.
- Jaina and Kael'thas in the year 603 bookmark won't have the Marrying Up modifier when marrying because they're related to the king and emperor tiers after all.
- Split Northsea culture from Southsea culture. While Southseans will be Africans, Northseans will look like Kul Tirans.
- Added Flynn Fairwind.
- Tidemother faith no longer has indian CoAs.
- Added Eimear, a wife of Magni.

# Love Changelog:
- Replaced many lover historical relationship by soulmates because soulmate shows the lover relationship shows a little affair while the soulmate is a lot more romantic. These are:
  - Griselda & Turok
  - Drum Fel & Korin
  - Moira & Dagran
  - Varian & Tiffin
  - Morelan Vanyst & Mary
  - Cairne & Tamaala
  - Pas'draen & Natalia
  - Tyrande & Malfurion
  - Jarod & Shalasyr
  - Alleria & Turalyon
  - Vereesa & Rhonin
  - Vynaen & Beledine
  - Daelin & Katherine
  - Jaina & Arthas
  - Arthur & Meredith Waycrests
  - Aldrius & Elena
  - Zhang & Mia
  - Tammis & Clannia
  - Zoltan & Lucina
  - Terenas & Lianne
  - Calia & Kiril
  - Gregor Agamand & Nissa
  - Alexandros Mograine & Elena
  - Ramnulf Thorval & Carolaine
  - Robert Garithos & Aliberta
  - Nathanos & Sylvanas
  - James Hyal & Lynn
  - Killoren & Vivian
  - Ysera & Eranikus
  - Deathwing & Sintharia
  - Alexstrasza & Tyranastrasz
  - Nozdormu & Soridormi
  - Luzik Gallywix & Lisa
  - Genn Greymane & Mia
  - Draka & Durotan
  - Tirion Fordring & Karandra
  - Malygos & Saragosa

<!--
A changelog where you can describe more complicated things to other developers and testers.
-->
## Developer changelog:
- Moved some triggers to the health file.
- Renamed `is_great_invader_trigger` to `is_crisis_trigger`.
- Moved `is_horde_invader_trigger` to the crisis file.

# Tasks
- [x] Add Shatter Realm game concept
- [x] Rename `wc_invasion` into `crisis` and update its info
- [x] Renamed `is_great_invader_trigger` into `is_crisis_trigger`

<!--
Before the merge, we recommend you to do these tests with your branch.
-->
## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:
To break the Union, right-click any member, **_DEBUG Interactions_**, **_Break Union_**.

To make a character undead, right-click them, **_DEBUG Interactions_**, **_Become Undead_**.

To make a wife pregnant, right-click her, **_DEBUG Interactions_**, **_Start Pregnancy_**.

To boost your murder schemes, type `yesman` in the console, it will force the AI to join your schemes.

ValianBlue commented:
_Some observations I've made._

_Jaina and Kael'thas sometimes wander away from Dalaran and do their own thing. I believe in Jaina's case, this depends on how the game determines her prowess at the start, which influences if she becomes a knight or not._

_"The Union has **not** been destroyed lately" is not specific. The date that the union was destroyed should be included in this interface to inform the player when it will be possible to create the union again._

<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/143780813-f8177157-ed8a-488c-8db9-3e3af67606a9.png)
</details>

_For hostile schemes, I'd consider reflavoring them. For example, in the "Knights and Peasants" scheme, the target is attacked by rioting peasants. While this would work perfectly for races like humans, elves, etc. it becomes a stretch for powerful beings like dragons. Instead, you could have it where the player is paying a party or band of adventurers to go and slay the dragon._

_Some relationships I think that could be added as soulmates._
- _Garona Halforcen & Medivh (first war)_
- _Mattias Shaw & Flynn Fairwind (would be later on in BfA lore, so this is low priority)_
- _Genn Greymane & Mia_
- _Kalecgos & Anveena Teague_
- _Magni Bronzebeard & Eimear (she dies "years later" after Moira is born)_
- _Draka & Durotan_
- _Malygos & Sindragosa, Saragosa, Haleh? (Not sure which one, maybe all three, heh)_
- _Tirion Fordring & Karandra_

_This is not a comprehensive list, just some of the ones that I could think of._

zumbak04 commented:
> "The Union has not been destroyed lately" is not specific. The date that the union was destroyed should be included in this interface to inform the player when it will be possible to create the union again.

Don't think it's possible. Instead, mentioned years you have to wait in 2e4dff93872ff2625da7e58e1310e99d52628f7b.

zumbak04 commented:
> While this would work perfectly for races like humans, elves, etc. it becomes a stretch for powerful beings like dragons.

I limited it to mammals and non-Dark governments in cce82eb774c4699972d04d63283303e29a7d0d3f.

> Instead, you could have it where the player is paying a party or band of adventurers to go and slay the dragon.

There's a similar event but with a band of thugs that attack you on road. So I guess it works.

zumbak04 commented:
> Kalecgos & Anveena Teague

She's not a living being, she is an avatar of the Sunwell thus she's not planned as a character.

The rest relationships are added.

TheGroxEmpire reviewed (commented):
- Shatter CB doesn't have declaration cost. I think there should be some Prestige cost on it.

- I can confirm that pregnancy is cancelled upon becoming Undead.

- I can't really confirm the murder plot to be working accordingly 100% but from several murder plots test, I found that Murloc and Dragon both only could have 3 outcomes on murder plot: Assassin, Accosted, A Cask of Wine.

zumbak04 commented:
> * Shatter CB doesn't have declaration cost. I think there should be some Prestige cost on it.

I may add, but really tiny one. You don't get any lands from the CB, you get or lose prestige. Also, it targets your possible threat like the Scourge, Horde, Legion, etc. I wouldn't lock defending them behind the paywall.

TheGroxEmpire reviewed (approved):
The rest looks good to me.

Holt228 reviewed (commented):
1) 
ER: Jaina and Kael'thas decide who to marry, not Antonidas due to being their liege.
AR: Jaina and Kael'thas decide who to marry (the tooltip shows that it's Jaina or Kael'thas' decision, not Antonidas')

2)
ER: Union members don't join civil wars.
AR: Union members cannot join civil wars (and the button to call them into the war is greyed out)

3)

ER: Arthas and Jaina prefer not to marry anyone for the first 5 years (to give them time)
AR: Arthas and Jaina usually don't marry anyone (did 10 tests total, 9 times both characters didn't marry anyone, 1 time Arthas married an NPC of a random culture)

4) 
ER: Becoming undead stops pregnancy
AR: Becoming undead gets rid of the pregnancy trait. No children are born.

5) 
ER: Demons, undead, amphibious and reptilian races can't be murdered by poison schemes
AR: Poison schemes do not show up when trying to assassinate demons, undead, amphibious and reptilian races 

6) 

ER: Dragons, harpies can't be murdered by fall accident schemes
AR: Was unable to test this one, will come back and do it at a later date

7) 
ER: Amphibious races, nagas can't be murdered by boat accident schemes
AR: Boat schemes do not show up when trying to assassinate amphibious races, nagas.

8) 
ER: To stay in the Union, Union Members have to pay a part of their income, levies, and suffer from high maintenance of MaAs
AR: Union members suffer penalties of about 10%

9) 
ER: The Union can't be created 10 years after it has been destroyed. Added the Break Union debug interaction.
AR: The Union can't be created 10 years after it has been destroyed. Break union debug interaction is present

10) 

ER: The Shatter Realm CB can be used against union leaders to destroy their unions.
AR: The Shatter Realm CB can be used against union leaders to destroy their unions.

11) 
ER: Scourge, Horde, Legion and other invasions happen at some time during gameplay.
AR: Scourge and Horde invasions reliably happen, haven't yet observed Legion or other invasions.

12) 
ER: Shatter Realm CB can be used against Union leaders
AR: Shatter Realm CB can be used against Union leaders

13) 
ER: Jaina and Kael'thas won't have the Marrying Up modifier when marrying
AR: Jaina has the Marrying up modifier when marrying, Kael'thas doesn't

14) 
ER: Southseans use African graphical culture, Northseans will look like Kul Tirans.
AR: Southseans indeed use African graphical culture, Northseans unse Kul Tiras graphical culture

15) 
ER: Flynn Fairwind can be found through character finder
AR:  Flynn Fairwind is present in-game

16) 
ER: Tidemother faith no longer has indian CoAs
AR: Tidemother faith uses random CoAs, doesn't use indian ones.

17) 
AR: Eimear shows up as a dead wife of Magni
ER: Eimear shows up as a dead wife of Magni

18) 
ER: The mod takes less than 5.5 GB in the Task Manager (Windows)
AR: The mod takes up 5.2 GB of RAM in the Task Manager

19)
ER: There are no errors in wc files in Documents\Paradox Interactive\Crusader Kings III\logs\error.log except portrait_decals.cpp:101
AR: There's also a following error: **[jomini_script_system.cpp:169]: Script system error!** in the following file **wc_special_wars.txt line: 13; shatter_realm_cb_valid line: 6; leads_any_pol_faction_trigger line: 5; leads_pol_faction_trigger line: 3**

zumbak04 commented:
> Scourge, Horde, Legion and other invasions happen at some time during gameplay.

Do you refer to the Crisis game concept description?

> AR: Jaina has the Marrying up modifier when marrying, Kael'thas doesn't

I suppose you marry them to a emperor?

zumbak04 commented:
> AR: There's also a following error: **[jomini_script_system.cpp:169]: Script system error!** in the following file **wc_special_wars.txt line: 13; shatter_realm_cb_valid line: 6; leads_any_pol_faction_trigger line: 5; leads_pol_faction_trigger line: 3**

Fixed in cc5f224aca1d1a239700f574d3aaff00c17a86a8.

Panabana reviewed (commented):
Hello! Here's a point list of things I tested:

1. Jaina and Kael'thas Marriage Decision
   • Jaina's modifiers shows that her liege cannot marry her away, as intended.
   • Kael'thas, I didn't look at sry.

2. Union members and Civil Wars
   • Working as intended.

3. Arthas and Jaina 5 Year Marriage Wait
   • 2/2 times Arthas married randomly after waiting properly.
   •  2/2 times Jaina married randomly after waiting properly.

4. Becoming Undead Stops Pregnancy
   • Tests proved pregnancies were cancelled upon using the undead DEBUG command.

5.  No Poison Schemes
   • Amphibious: 2/2 murder plot tests worked as intended.
   • Undead: 2/2 murder plot tests worked as intended.
   • Demons: 1/1 murder plot tests worked as intended.
   • Reptilian: 2/2 murder plot tests worked as intended.

6. No Fall Accident Schemes
   • Dragons: 2/2 worked as intended.
   • Harpies: 1/1 worked as intended.

7. No Boat Accident Schemes
   • Amphibious: 2/2 worked as intended.
   • Nagas: 1/1 worked as intended.

8. Union Member Payment
   • As intended, union members pay 10% income, have increased 20% MaA maintenance and and decreased 40% levy size.
   • Union leader gets both effects of "Union Leader" and "Union Member", is that intended? Leaders shouldn't be penalized with the "member" debuffs right? Logically, it seems like they are the ones receiving from the members.

9. Create/Break Union
   • "Break Union" debug interaction available and working as intended.
   •  The 10 year cooldown is working as intended.

10. Shatter Realm CB Destroys Union
   • Shatter Realm works just like mentioned, vassals are freed and union is destroyed.

11. Crisis tests (Only as observer)
   • Scourge didn't happen as observer.
   • Horde happens, coz they start at the first start date anyways though? Is it possible for the Horde to spawn in too?
   • Legion didn't happen as observer.
   • I think I misunderstood this point, sry in advance :))

12. Using Shatter Realm CB
   • Confirmed. It can be used against Union Leaders.

13. Jaina and Kael'Thas no Marrying Up Modifier
   • Found no mistakes here.

14. Northseans and Southseans
    • Did not confirm.

15. Flynn
   • Flynn Fairwind was found through the character finder.

16. Tidemother faith no longer Indian CoAs
   • Where do I find the Tidemother faith? I looked at the lists but failed to find them, sooorry.

17. Eimear
   • Confirmed. Eimear shows up as a dead wife of Magni


Script system errors (Just a couple out of twenty. Don't know if these are useful info tho)

• Error: scope:faction.faction_target trigger [ Failed context switch ]
  Script location: file:  file: common/factions/00_peasant_faction_new.txt line: 204; scourge_county_rebel_modifiers line: 22

• Error: scope:faction.faction_target trigger [ Failed context switch ]
  Script location: file:  file: common/factions/00_factions.txt line: 274; scourge_character_rebel_modifiers line: 22

/Pana

zumbak04 commented:
> Union leader gets both effects of "Union Leader" and "Union Member", is that intended? Leaders shouldn't be penalized with the "member" debuffs right? Logically, it seems like they are the ones receiving from the members.

No, you pay for keeping alliances.

> Is it possible for the Horde to spawn in too?

Our first bookmark has the Horde already.

> I think I misunderstood this point, sry in advance :))

Yes, it's just a text with descriptions. You need to read it, that's all.

> Did not confirm.

Why?

> Where do I find the Tidemother faith? I looked at the lists but failed to find them, sooorry.

Kul Tiras

> Script system errors (Just a couple out of twenty. Don't know if these are useful info tho)

Yes, we know about them.

zumbak04 commented:
Test

Panabana commented on localization/english/wc_effects_l_english.yml:
```diff
@@ -80,4 +80,16 @@
+ raise_undead_by_vassal_effect_global_past:1 "Became [GetTrait('being_undead').GetName(GetNullCharacter)|V] and converted to [TARGET_CHARACTER.GetFaith.GetName|V]"
+ 
+ break_union_effect_global:0 "The [Concept('union', CHARACTER.Custom( 'GetPolFactionName' ))|E] falls apart"
+ break_union_effect_global_past:0 "The [Concept('union', CHARACTER.Custom( 'GetPolFactionName' ))|E] falled apart"
```
```suggestion
 break_union_effect_global_past:0 "The [Concept('union', CHARACTER.Custom( 'GetPolFactionName' ))|E] fell apart"
```

Panabana commented on localization/english/wc_game_concepts_l_english.yml:
```diff
@@ -29,9 +29,9 @@
- game_concept_wc_invasion_desc:0 "A very expansive power in the world. Getting almost free [casus_bellis|E] and increased aggressiveness, they need special attention from more peaceful rulers. Having an $game_concept_ongoing_invasion$ activates more options to defend such as [unions|E].\n\nThere may exist these invasions:\n$EFFECT_LIST_BULLET$$e_horde$\n$EFFECT_LIST_BULLET$$e_scourge$\n$EFFECT_LI ...
+ game_concept_crisis:1 "Crisis"
+ game_concept_ongoing_crisis:1 "Ongoing Crisis"
+ game_concept_crisis_desc:1 "A very expansive power in the world.\n\nGetting almost free [casus_bellis|E] like [great_invasion_casus_belli|E] and increased aggressiveness, they need special attention from more peaceful rulers.\n\nHaving an $game_concept_ongoing_crisis$ activates more options to defend such as [unions|E] or using [shatter_realm_casus_belli|E] against them.\n\nThere may exist these ...
```
```suggestion
 game_concept_crisis_desc:1 "A colossal power in the world.\n\nThey gain almost free [casus_bellis|E], like the  [great_invasion_casus_belli|E], and increased aggressiveness. They need special attention from more peaceful rulers.\n\nHaving an $game_concept_ongoing_crisis$ activates further options to defend oneself, such as [unions|E] or using the [shatter_realm_casus_belli|E] against them.\n\nThere can exist these $game_concept_crises$:\n$EFFECT_LIST_BULLET$$e_horde$\n$EFFECT_LIST_BULLET$[scourge|E]\n$EFFECT_LIST_BULLET$$e_burning_legion$\n$EFFECT_LIST_BULLET$And more..."
```

Panabana commented on localization/english/wc_game_concepts_l_english.yml:
```diff
@@ -83,10 +83,10 @@
  game_concept_scourge:0 "Scourge"
  game_concept_scourge_possessive:0 "Scourge's"
- game_concept_scourge_desc:0 "One of the main threats on Azeroth. The $game_concept_scourge$ has a [necro_government|E] and is led by the [GetTrait('lich_king').GetName( GetNullCharacter )], a powerful necromancer who controls their [vassals|E] with an iron grip.\n\nIt enables [great_invasion_casus_belli|E] but instead of taking lands you take vassals and turn them [GetTrait('being_undead').GetNa ...
+ game_concept_scourge_desc:1 "One of the main [crises|E] on Azeroth. The $game_concept_scourge$ has a [necro_government|E] and is led by the [GetTrait('lich_king').GetName( GetNullCharacter )], a powerful necromancer who controls their [vassals|E] with an iron grip.\n\nIt enables [great_invasion_casus_belli|E] but instead of taking lands you take vassals and turn them [GetTrait('being_undead').Ge ...
```
```suggestion
 game_concept_scourge_desc:1 "One of the main [crises|E] on Azeroth. The $game_concept_scourge$ has a [necro_government|E] and is led by the [GetTrait('lich_king').GetName( GetNullCharacter )], a powerful necromancer who controls their [vassals|E] with an iron grip.\n\nThe $game_concept_scourge$ has the [great_invasion_casus_belli|E] ,but instead of taking lands it takes vassals and turn them [GetTrait('being_undead').GetName( GetNullCharacter )].\n\nThe $game_concept_scourge$ also starts as a [legions_puppet|E], and must spend the first 15 years dedicated to bringing the Legion to Azeroth."
```

Panabana commented on localization/english/wc_game_concepts_l_english.yml:
```diff
@@ -98,4 +98,21 @@
+ game_concept_shatter_realm_casus_belli:0 "Shatter Realm Casus Belli"
+ game_concept_shatter_realm_casus_belli_desc:0 "A powerful [casus_belli|E] available against [crises|E] and [union_leaders|E]. As a result, the $shatter_realm_casus_belli_common_info$."
+ 
+ shatter_realm_casus_belli_common_info:0 "realm falls apart freeing [vassals|E], giving lands to [claimants|E] and leaving or breaking a [union|E]"
```
```suggestion
 shatter_realm_casus_belli_common_info:0 "realm falls apart freeing [vassals|E], giving lands to [claimants|E] and leaves or breaks a [union|E]"
```

Panabana commented:
Okay I checked out the texts as requested and done my suggestions, rest looks great!

I also figured out and checked out North/Southsea and they look as intended.
Same with Tirani/Tidewater faith no longer having Indian CoAs.

Aside from my suggestions everything LGTM!

/Pana

MemeWizard commented on localization/english/wc_game_concepts_l_english.yml:
```diff
@@ -98,4 +98,21 @@
+ game_concept_shatter_realm_casus_belli:0 "Shatter Realm Casus Belli"
+ game_concept_shatter_realm_casus_belli_desc:0 "A powerful [casus_belli|E] available against [crises|E] and [union_leaders|E]. As a result, the $shatter_realm_casus_belli_common_info$."
+ 
+ shatter_realm_casus_belli_common_info:0 "realm falls apart freeing [vassals|E], giving lands to [claimants|E] and leaves or breaks a [union|E]"
```
```suggestion
 shatter_realm_casus_belli_common_info:0 "realm falls apart freeing [vassals|E], giving lands to [claimants|E] and defeated [character|E] leaves or breaks a [union|E]"
```

MemeWizard commented on localization/english/modifiers/wc_modifiers_l_english.yml:
```diff
@@ -65,4 +65,7 @@
  plague_watchful_modifier:0 "Very Watchful"
- plague_watchful_modifier_desc:0 "$watchful_modifier_desc$"
\ No newline at end of file
+ plague_watchful_modifier_desc:0 "$watchful_modifier_desc$" long_lived_lifestyle_xp_slowdown_custom_desc:0 "$game_concept_knowledge_overflow_common_desc$\n\nDue to [ROOT.Char.GetShortUINamePossessive] [life_expectancy|E], [lifestyle_experience|E] slow downs by #N [GuiScope.SetRoot(ROOT.Char.MakeScope).ScriptValue('long_lived_lifestyle_xp_slowdown_per_perk_percent_value')|2]%#! per each [perk|E] s ...
```
Doubled line
```suggestion
 plague_watchful_modifier_desc:0 "$watchful_modifier_desc$"
```

MemeWizard commented on localization/english/wc_effects_l_english.yml:
```diff
@@ -80,4 +80,16 @@
+ break_union_effect_global:0 "The [Concept('union', CHARACTER.Custom( 'GetPolFactionName' ))|E] falls apart"
+ break_union_effect_global_past:0 "The [Concept('union', CHARACTER.Custom( 'GetPolFactionName' ))|E] fell apart"
+
+ leave_union_tooltip_first:0 "You leave the current [union|E]"
+ leave_union_tooltip_first_past:0 "You left the current [union|E]"
+ 
+ leave_union_tooltip_third:0 "[CHARACTER.GetShortUIName|U] leaves the current [union|E]"
+ leave_union_tooltip_third_past:0 "[CHARACTER.GetShortUIName|U] left the current [union|E]"
```
I suggest to put them in the same file for the sake of convenience

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/154790863-80c1af6e-cb78-4bd4-ba23-73843e816e37.png)


</details>

MemeWizard replied:
Also, it has the same problem - since you speak of it in the past tense, it's not longer "current" union.

zumbak04 replied:
Fixed in bd96e8aba6d3197bd326bb1cefaf5d4cfd25d734.

MemeWizard commented on localization/english/wc_character_interactions_l_english.yml:
```diff
@@ -13,13 +13,13 @@
- OWNER_FAITH_SINS:0 "Disapproves [owner.GetFaith.GetAdjective|V] [sins|E]: $VALUE|=+0$"
\ No newline at end of file
+ OWNER_FAITH_SINS:0 "Disapproves [owner.GetFaith.GetAdjective|V] [sins|E]: $VALUE|=+0$" RECIPIENT_SCOURGE_JAILOR_REASON:1 "[recipient.GetShortUIName|U] is a [scourge_jailor|E]: $VALUE|=+0$"
+ MARRIAGE_RECIPIENT_HAS_SOMEONE_ELSE:1 "[secondary_recipient.GetShortUIName|U] has someone else in mind: $VALUE|=+0$"
+ MARRIAGE_ACTOR_HAS_SOMEONE_ELSE:1 "[secondary_actor.GetShortUIName|U] has someone else in mind: $VALUE|=+0$"
```
```suggestion
 MARRIAGE_RECIPIENT_HAS_SOMEONE_ELSE:1 "[secondary_recipient.GetShortUIName|U] has [Concept('soulmate','someone')|E] else in mind: $VALUE|=+0$"
 MARRIAGE_ACTOR_HAS_SOMEONE_ELSE:1 "[secondary_actor.GetShortUIName|U] has [Concept('soulmate','someone')|E] else in mind: $VALUE|=+0$"
```

MemeWizard reviewed (commented):
> Jaina and Kael'thas decide who to marry, not Antonidas due to being their liege.

This changes recipient from Antonidas to their courtier, thus making "clear character" button behave in a unexpected way - returning you to that courtier instead of Antonidas.

So, how about add this check

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/154790181-72854925-d546-4684-9719-1f6f4075086d.png)

</details>

before adding characters in `populate_recipient_list`?

Imo, if recipient changes to the potential future spouse, then it would be better if Antonidas didn't had his courtiers in that list, to avoid confusion caused by redirection.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/154788467-87a2b9ec-1a18-46f4-a20f-8e0e79ee63b7.png)
![image](https://user-images.githubusercontent.com/46576860/154788472-0205ac65-8c3c-4283-bd34-0f3c31a368f7.png)
![image](https://user-images.githubusercontent.com/46576860/154788482-4eabde8f-bc9a-46a1-bdeb-ef9b14042c36.png)
![image](https://user-images.githubusercontent.com/46576860/154788484-d3cff1f9-4a4c-4dfc-954c-d3e304db4ad5.png)

</details>

> Union members don't join civil wars.

Please, add that important info in union concept.

zumbak04 commented:
> This changes recipient from Antonidas to their courtier, thus making "clear character" button behave in a unexpected way - returning you to that courtier instead of Antonidas.

Changed in 58f8e559a6f2a96513d193946f6e4c19bcd3b77b.

> Please, add that important info in union concept.

Added in f848d09f00d88d48fa0496f4626ad637ea3ee187.

Caitqua reviewed (commented):
-	If you are pregnant before becoming undead, you are still pregnant after becoming undead, however it seems that the pregnancy doesn't progress? I think that this confuses things a little bit, as my zombie queen of Zandalar has been pregnant for, something like 23 years.
-	11 years in one game, Jaina was betrothed to the chancellor of Balor, off the coast of Stormwind, at the age of 14. She later got a venereal disease at age 18.
-	After murdering every dragon, demon, and naga in the world, I have come to the conclusion that the fatality rate of weak balconies, bad cooks, and water have fallen to an all time low of 0%.
-	It would appear that all of the Union changes are working as intended.
-	It looks like the Shatter Realm CB is working as intended, as long as it is only intended to break up a union. Petition to rename it "Shatter Union?"
-	Jaina still thinks she's marrying up.
-	Found both the Northsea and Southsea cultures, both seem to be WAI.
-	Found Flynn
-	It looks like the Tidemother does indeed no longer have indian CoAs.
-	Found Eimear!

SUGGESTIONS:
-	Make it so that events that grant the Undead trait remove the Pregnant trait.

zumbak04 commented:
> * If you are pregnant before becoming undead, you are still pregnant after becoming undead, however it seems that the pregnancy doesn't progress? I think that this confuses things a little bit, as my zombie queen of Zandalar has been pregnant for, something like 23 years.

Can't confirm. Maybe you forgot to switch the branch?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/155877046-731fc186-24dd-47dc-b340-ade66216a2d4.png)
![image](https://user-images.githubusercontent.com/46384992/155877054-e572cc49-a198-40f7-85f3-78bb6d5f6db6.png)

</details>

> 11 years in one game, Jaina was betrothed to the chancellor of Balor, off the coast of Stormwind, at the age of 14. She later got a venereal disease at age 18. Not sure if this is her choice or her liege's.

Don't get what you tested.

> It looks like the Shatter Realm CB is working as intended, as long as it is only intended to break up a union. Petition to rename it "Shatter Union?"

No, it's also used against the Horde and Scourge.

> Jaina still thinks she's marrying up when I played as Arthas.

Arthas is a relative of an emperor, Jaina - of a king.

zumbak04 commented on common/character_interactions/00_marriage_interactions.txt:
```diff
@@ -73,6 +73,11 @@ arrange_marriage_interaction = {
 				limit = {
```
> You can select Kael'thas in Anasterian court, but then it redirects recipient to Kael'thas, breaking character clearing.

You always could. Except it would redirect to Antonidas.

zumbak04 replied:
> Also, it would be nice to give governments that can't be matchmakers for their courtiers a tooltip about it, like these.

Added in 4e077e557663f63a80e35943b1819015b9628525.

TheDrPibb1999 reviewed (approved):
Went through list, LGTM - merge her!

ValianBlue reviewed (approved):
_Original review: [issuecomment-981135947](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/775#issuecomment-981135947)._

_All changes look good, nicely done!_
